CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -70,6 +73,10 @@ local ERROR_ICON = 'rbxasset://textures/ui/ErrorIcon.png'
 local A_BUTTON = "rbxasset://textures/ui/Settings/Help/AButtonDark.png"
 local B_BUTTON = "rbxasset://textures/ui/Settings/Help/BButtonDark.png"
 local DEFAULT_XBOX_IMAGE = 'rbxasset://textures/ui/Shell/Icons/ROBUXIcon@1080.png'
+local VR_BUTTON = 'rbxasset://textures/ui/VR/button.png'
+local VR_BUTTON_DOWN = 'rbxasset://textures/ui/VR/buttonSelected.png'
+local emptySelectionImage = Instance.new("ImageLabel")
```

Comment:
Even though this Selection Image is supposed to be just a stand in we should give it a name for debugging purposes